articles/quickstart/native/ios-objc/03-user-sessions.md

```diff
@@ -234,11 +234,11 @@ You need to call a method from the `Auth0` module that allows you to fetch the u
 }];
 ```
 
-## Show User Profile's Data
+## Show the User Profile Information
 
-#### Default info
+### Default information
```

Comment:
Why is `information` not capitalised but in the header above it is?

Reply:
According to the style guide, H2 should be in title case and H3 should be in sentence case :)